lib/log.rb

```diff
@@ -158,9 +158,8 @@ def print(msg)
 
     # supports embedding content similar with same semantics as Cucumber
     def embed(src, mime_type, label)
-      if self.class.runtime.respond_to? :embed
-        info "embedding #{label.inspect}"
-        self.class.runtime.embed src, mime_type, label
+      if self.class.runtime.respond_to? :attach
+        self.class.runtime.attach src, mime_type
```

Comment:
@pruan-rht Thank you for the fix.
@yapei  Could you help to test the changes ? Thanks.

Reply:
@liangxia I tested locally by using a bad xyaml...the screen capture took place and the html is saved.